src/windows/export.py

```diff
@@ -732,6 +740,7 @@ def accept(self):
             w.Open()
 
             # Notify window of export started
+            export_file_with_path = export_file_path
```

Comment:
I think we need to remove this line, and the QMessageBox code block below, so we don't introduce a new translatable element this late (fixing to release very soon). I'm also not 100% sure about the idea of a pop-up at the end of an export either. Pehaps this could be brought back as a debug preference later?